spectrum-network/src/peer_manager.rs

```diff
@@ -491,9 +515,11 @@ impl<S: PeersState> PeerManagerNotificationsBehavior for PeerManager<S> {
         }
     }
 
-    fn on_connection_established(&mut self, peer_id: PeerId, _conn_id: ConnectionId) {
+    fn on_connection_established(&mut self, peer_id: PeerId, conn_id: ConnectionId) {
         if let Some(PeerInState::Connected(mut cp)) = self.state.peer(&peer_id) {
             cp.confirm_connection();
+            self.out_queue
+                .push_back(PeerManagerOut::EstablishOutgoingConnection(peer_id, conn_id))
```

Comment:
Connection is already established and PM notified about that in the last place, why issuing new command?

Reply:
There's no reason to issue it, thanks.